test/versions.bats

```diff
@@ -161,3 +169,34 @@ OUT
   run pyenv-versions --bare
   assert_success ".venv"
 }
+
+@test "uses version sort when available" {
+  create_version "1.9.0"
+  create_version "1.53.0"
+  create_version "1.218.0"
+
+  correct_versions_file="$(mktemp)"
```

Comment:
Any temporary files need to be created under `$PYENV_TEST_DIR` so that they are deleted after the test. Does this ensure that?

Reply:
It doesn't -- I thought about that after I pushed the commit, I'll fix it.